example/resources/users.py

```diff
@@ -71,3 +71,13 @@ def replace(self, params):
             'href': new_item['photos'].url()
         }
         return user
+
+
+@royal.collection_config('users.followers')
+class UserFollowers(royal.Collection):
+    pass
+
+
+@royal.item_config('users.followers')
+class Follower(Item):
+    pass
```

Comment:
These two class definitions, in particular their relationship with their superclass,) are quite confusing. I guess that `Follower` is a kind of `user.Item`, but it took me a while to find this out and I'm not even sure.